Dockerfile.local

```diff
@@ -3,6 +3,7 @@ ENV LANG C
 
 COPY dnf-docker-test/repo /var/www/html/repo/
 COPY dnf-docker-test/features /behave/
+COPY dnf-docker-test/features /tests/
```

Comment:
```COPY dnf-docker-test/features /behave/```
Here I need to copy everything except *.feature files (dirs and .*py files)
```COPY dnf-docker-test/features /tests/```
Here I need to copy *.feature files